docs/resources/node_pool.md

```diff
@@ -65,6 +65,7 @@ The following arguments are supported:
 * `hostname_prefix` - (Required) The prefix for created nodes of the Node Pool (string)
 * `node_template_id` - (Required) The Node Template ID to use for node creation (string)
 * `delete_not_ready_after_secs` - (Optional) Delete not ready node after secs. For Rancher v2.3.3 or above. Default `0` (int)
+* `drain_before_delete` - (Optional) Drain nodes before delete (bool)
```

Comment:
what is the default behavior?

Reply:
All tf bool without explicit default value are treat as `false`. Do you prefer to be explicit here?

Reply:
@paynejacob Updated docs being explicit with `drain_before_delete` default value